tasks/main.yml

```diff
@@ -137,17 +141,42 @@
   when: python_deploy__type == "pypi"
   notify: '{{ python_deploy__combined_notify if python_deploy__combined_notify|d() else omit }}'
 
-- name: 'Install requirements'
+- name: Install requirements with pip
   pip:
     requirements: '{{ python_deploy__path_app }}/{{ item }}'
     virtualenv: '{{ python_deploy__path_venv }}'
     virtualenv_python: '{{ python_deploy__python }}'
   become: yes
   become_user: '{{ python_deploy__user }}'
   with_items: '{{ python_deploy__requirements_files }}'
-  when: python_deploy__type == "git_without_setup"
+  when: python_deploy__type == "git_without_setup" and not python_deploy__use_poetry
```

Comment:
Jak przyjmiemy nowy wariant dla zmiennej `python_deploy_type` to trzeba to zmienić

Reply:
A chcemy dla innych typów instalacji używać Poetry? Jak to powinno wyglądać dla "git_with_setup"?